runtime/altair/src/xcm.rs

```diff
@@ -62,8 +62,19 @@ pub type Trader = (
 	FixedRateOfFungible<AirPerSecond, ToTreasury>,
 	FixedRateOfFungible<KUsdPerSecond, ToTreasury>,
 	FixedRateOfFungible<KsmPerSecond, ToTreasury>,
+	AssetRegistryTrader<FixedRateAssetRegistryTrader<FixedConversionRateProvider>, ToTreasury>,
 );
 
+pub struct FixedConversionRateProvider;
+
+impl orml_traits::FixedConversionRateProvider for FixedConversionRateProvider {
+	fn get_fee_per_second(location: &MultiLocation) -> Option<u128> {
+		let metadata = OrmlAssetRegistry::fetch_metadata_by_location(&location)?;
+		// TODO(nuno): discuss internally
```

Comment:
`todo` We need to agree on this. Alternatively, add a `fee_per_second` field to the `CustomMetadata` so that it's not hardcoded.